From d5d71b5dfffeda33591f6f82dbacd32296f78d1b Mon Sep 17 00:00:00 2001
From: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Date: Tue, 3 Dec 2019 01:03:54 -0800
Subject: [PATCH 0068/1852] firmware: xilinx: Add IOCTL for handling usb power
 requests

commit f177a992daf08eaec5463276a4472b18df6daf0e from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the IOCTL_USB_SET_STATE enum into firmware
pm_ioctl_id for requesting U2PMU to handle usb power states.

Signed-off-by: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Rajan Vaja <rajan.vaja@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/linux/firmware/xlnx-zynqmp.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/firmware/xlnx-zynqmp.h b/include/linux/firmware/xlnx-zynqmp.h
index d6cc1a9a5e87..89561f00a3b4 100644
--- a/include/linux/firmware/xlnx-zynqmp.h
+++ b/include/linux/firmware/xlnx-zynqmp.h
@@ -171,6 +171,8 @@ enum pm_ioctl_id {
 	IOCTL_PROBE_COUNTER_READ,
 	IOCTL_PROBE_COUNTER_WRITE,
 	IOCTL_OSPI_MUX_SELECT,
+	/* IOCTL for USB power request */
+	IOCTL_USB_SET_STATE,
 };
 
 enum pm_query_id {
-- 
2.31.1

